views/pages/profile/index.ejs

```diff
@@ -0,0 +1,20 @@
+<div class="card">
+  <div class="card-header">
+    <h1 class="text-center">User Information</h1>
+  </div>
+  <div class="card-body">
+    <h2>Name:</h2>
+  </div>
+  <hr />
+  <div class="card-body">
+    <h2>User ID:</h2>
+  </div>
+  <hr />
+  <div class="card-body">
+    <h2>Email:</h2>
```

Comment:
Could add in <%= email %> since we have that data

Reply:
This was added from your recommendation in commit 3c6e154 thanks!